lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -351,6 +368,35 @@ def add_elasticsearch_index_name_field(tag, time, record)
       end
     end
 
+    def transform_event(tag, record)
+      return if @event_trans.empty?
+      return if @event_trans_cache_nomatch[tag]
+      unless @event_trans_matcher.match(tag)
+        @event_trans_cache_nomatch[tag] = true
+        return
+      end
+
+      if @event_trans_type == :eventrouter
+        if record.has_key?("event")
+          if record.has_key?("verb")
+            record["event"]["verb"] = record.delete("verb")
+          end
+          if record["event"].has_key?("message")
+            record["pipeline_metadata"] = {} unless record.has_key?("pipeline_metadata")
+            record["pipeline_metadata"]["collector"] = {} unless record["pipeline_metadata"].has_key?("collector")
+            record["pipeline_metadata"]["collector"]["original_raw_message"] = record["message"]
```

Comment:
Is this already being performed in `add_pipeline_metadata`?

Reply:
yes

Reply:
Is there any other `add_pipeline_metadata` than this one?
https://github.com/ViaQ/fluent-plugin-viaq_data_model/blob/cc806f964ed937ed84c842c498a0913a6a1a2470/lib/fluent/plugin/filter_viaq_data_model.rb#L281-L290

I can't seem to find it there and I am not sure what am I missing. Could you please explain?

Reply:
What we mean is that we already have a function to add pipeline metadata, and you are duplicating that method in your code.

Reply:
Oh, ok. Got it, thanks! :)